### DISCUSSION TITLE
Blocking scripts without data-category

### DISCUSSION THREAD

**Discussion by ehilber** [orestbida/cookieconsent#708] [General]
Hi! I'm implementing CookieConsent 3.0.1. and use the following tag-manager code. 

<script defer>
	(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
	new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
	j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
	'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
	})(window,document,'script','dataLayer','GTM-XXXXXXX');
</script>

I have not yet added following attributes (according to https://cookieconsent.orestbida.com/advanced/manage-scripts.html):
    type="text/plain"
    data-category="analytics">

The following requests are fired: 
![grafik](https://github.com/user-attachments/assets/e289e3d2-7c44-4722-baf1-b35539e95598)
The following cookies are set:
![grafik](https://github.com/user-attachments/assets/70d8d1de-fdc5-4334-87dc-14d1e18942ee)

After accepting cookies for statistics, some further request are fired:
![grafik](https://github.com/user-attachments/assets/f5bccc2c-1c93-4c96-b70c-dfc02434a146)
Only now after accepting cookies for statistics the cookies for Google Analytics are set:
![grafik](https://github.com/user-attachments/assets/2a93ac39-0d9b-4fb2-9fcd-bc07e95464c1)

Does the script blocks www.googletagmanager.com/gtm.js?i... although the category is not specified? A script without any data-category should not be blocked right?

**Comment by orestbida** [comment]
No, the only way to disable a script is by adding an invalid javascript mime type, such as `type="text/plain"`, before the script tag is appended to the DOM. There is probably something else going on.
